Documentation/workflow/UnitTests.md

```diff
@@ -0,0 +1,267 @@
+# Unit Tests
+
+
+## Tools
+
+1. [NUnit](https://github.com/nunit/nunit)
+2. [XUnit](https://github.com/xunit/xunit)
+
+## Project Test Types
+
+1. Build Tests
+
+    There are a number of `Build` related tests which test various aspects of the `.NET Android` build system. This includes not only MSBuild Task tests but build integration tests. Since the SDK is consumed from MSBuild, almost all of the unit tests are MSBuild related.
```

Comment:
Style-wise, I don't think we need to use "code" formatting for ".NET Android".  It's not code, it's not an identifier, it's the name of our product.